EZSwiftExtensionsTests/ArrayTests.swift

```diff
@@ -285,6 +292,54 @@ class ArrayTests: XCTestCase {
         XCTAssertEqual(duplicatedElementsArrayOrderMaintained.unique(), [2, 1, 3])
     }
     
+    func testStep() {
+        XCTAssertNil(self.numberArray.step(-1))
+        XCTAssertNil(self.numberArray.step(0))
+
+        XCTAssertEqual(self.numberArray.step(2)!.count, 4)
+        XCTAssertEqual(self.numberArray.step(2)![0].count, 2) // count in each of the first child element will be <= stepSize
```

Comment:
In hindsight the count verifications are superflous, since we are also checking the subarray.